src/cljs/main/broadfcui/endpoints.cljs

```diff
@@ -851,7 +851,10 @@
                 "Aborted" (rand-int 1000)
                 "Unknown" (rand-int 1000)}}})
 
-(defn cromwell-version []
-  {:path "/version/executionEngine"
-   :method :get
-   :mock-data {"cromwell" "25-489f66b"}})
+(defn cromwell-version [on-done]
+  (utils/ajax-orch
+   "/executionEngine"
+   {:method :get
+    :on-done on-done}
+   :service-prefix "/version"))
```

Comment:
this is sort of artificial, using "/version" as the service prefix. but without a value for it, "/api" is appended. i could change the ajax-orch call instead.

Reply:
I think this is precisely how this is intended to be used

Reply:
there's precedent in `profile-get`, `profile-set`, too.